pkg/shell/plot.js

```diff
@@ -160,16 +165,33 @@ shell.plot = function plot(element, x_range) {
         walk_timer = null;
     }
 
-    function reset() {
+    function reset(x_range_seconds, x_stop_seconds) {
         stop_walking();
         for (var i = 0; i < data.length; i++)
             data[i].stop();
 
+        // XXX - breaks with DST change during x_range.
```

Comment:
Why are we dealing with local time at all for calculations? Shouldn't we just convert to local time immediately when displaying the data?